### PR TITLE
COM-6605 : Email template for reactivate subscription reminder

### PR DESCRIPTION
**Pull request by kibo-shrutikdeshmukh**

### CODE REVIEW

File: labels/en-US.json [hunk 1]
```diff
@@ -811,14 +811,15 @@
...
   "subscriptionDetails": "Subscription Details",
   "subscriptionErroredBlob1": "Hello, <br/><br/> The status of your subscription #{0} is updated to {1}.  Subscription has been moved to an errored state because of a possible issue with the payment or another issue. Please contact customer care for more details. If you would like to view or manage your subscription, please visit <a href=\"https://{2}/myaccount\">My Account</a>.",
   "subscriptionFrequency": "Frequency",
   "subscriptionFrequencyChangedBlob": "Hello, <br/><br/> Thank you for shopping with us at {0}. Your subscription was updated. If you would like to view or manage your subscription, please visit My Account.",
   "subscriptionFrequencyNotFound": "No matching subscription frequencies found",
+  "subscriptionReactivatedReminderBlob": "You have reached your pause limit and your subscription will be reactivated on {0}. If you would like to view or manage your subscription, please visit <a href=\"https://{1}/myaccount\">My Account</a>.",
```
Comment (atul-chavan) [inline labels/en-US.json]: plase arrange the label with alphabetical order

File: labels/de-DE.json [hunk 2]
```diff
@@ -689,6 +689,7 @@
   "subscriptionItemAddedBlob": "Hallo, <br/><br/> Vielen Dank für Ihren Einkauf bei uns unter {0}. Ihr Abonnement wurde aktualisiert. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, bitte besuche Mein Konto.",
   "subscriptionItemQuantityUpdatedBlob": "Hallo, <br/><br/> Vielen Dank für Ihren Einkauf bei uns unter {0}. Ihr Abonnement wurde aktualisiert. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, bitte besuche Mein Konto.",
   "subscriptionItemRemovedBlob": "Hallo, <br/><br/> Vielen Dank für Ihren Einkauf bei uns unter {0}. Ihr Abonnement wurde aktualisiert. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, bitte besuche Mein Konto.",
+  "subscriptionReactivatedReminderBlob": "Vous avez atteint votre limite de pause et votre abonnement sera réactivé le {0}. Si vous souhaitez consulter ou gérer votre abonnement, rendez-vous sur <a href=\"https://{1}/myaccount\">Mon compte</a>.",
```
Comment (atul-chavan) [inline labels/de-DE.json]: plase arrange the label with alphabetical order

**Review by atul-chavan — COMMENTED** [review]
plase rearrange the label with alphabetical order

File: labels/en-US.json (hunk 1 above)
Comment (kibo-shrutikdeshmukh) [reply labels/en-US.json]: Done

File: labels/de-DE.json (hunk 2 above)
Comment (kibo-shrutikdeshmukh) [reply labels/de-DE.json]: Done

File: labels/en-US.json (hunk 1 above)
Comment (atul-chavan) [inline labels/en-US.json]: subscriptionReactivationReminderBlob

File: labels/de-DE.json (hunk 2 above)
Comment (atul-chavan) [inline labels/de-DE.json]: subscriptionReactivationReminderBlob

File: templates/email/subscription-reactivation-reminder.hypr [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{% extends "email/email" %}
+{% block body-content %}
+
+ <!--- Header --->
+
+
+  <div class="mz-b2b-email-body">
+      {{ labels.emailOpeningAlternative|safe }}
+      {% with model.ReactivationDate|date('M d, Y') as ReactivationDate %}
+        <p>{{ labels.subscriptionReactivatedReminderBlob|string_format(ReactivationDate, domainName)|safe}}</p>
```
Comment (atul-chavan) [inline templates/email/subscription-reactivation-reminder.hypr]: subscriptionReactivationReminderBlob

Comment (kibo-shrutikdeshmukh) [reply templates/email/subscription-reactivation-reminder.hypr]: Done

File: labels/de-DE.json (hunk 2 above)
Comment (kibo-shrutikdeshmukh) [reply labels/de-DE.json]: Done

File: labels/en-US.json (hunk 1 above)
Comment (kibo-shrutikdeshmukh) [reply labels/en-US.json]: Done

**Review by atul-chavan — APPROVED** [review]

**Pull request merged**